tests/migrations/test_loader.py

```diff
@@ -464,3 +465,15 @@ def test_loading_squashed_ref_squashed(self):
             ('app1', '4_auto'),
         }
         self.assertEqual(plan, expected_plan)
+
+    def test_readonly_database(self):
+        """
+        MigrationLoader.check_consistent_history() shouldn't choke on
+        read-only databases (possibly without a django_migrations table).
+        """
+        with mock.patch.object(
+                MigrationRecorder, 'ensure_schema',
+                autospec=True,
```

Comment:
I think you're talking about the other test. This one passes and fails without `autospec`.